src/index.html

```diff
@@ -9,8 +9,64 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="images" href="images/Search.svg">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="http://localhost:8000/api"
+    method="post"
+    data-qa="big"
+    >
```

Comment:
```suggestion
    <form 
      action="http://localhost:8000/api"
      method="post"
      data-qa="big"
    >
```
Let's work on code style everywhere

Reply:
Done - ready for re-review.